test/integration/prepare_int_test.go

```diff
@@ -92,7 +92,7 @@ func (suite *PrepareIntegrationTestSuite) AssertConfig(target string) {
 		suite.Require().NoError(err)
 
 		subshell := subshell.New(cfg)
-		rcFile, err := subshell.RcFile()
+		rcFile, err := subshell.RcFile(false)
```

Comment:
I'm seeing a lot of churn in this diff related to the function signature change. I wonder if it may be more advantageous to have a new function specifically for creating the file if it already exists. (e.g. `subshell.RcFileEnsureExists()`) I'm just thinking out loud though. Not sure if this will help or just cause more churn elsewhere.

Reply:
That's a good point. After looking at this again I was able to get rid of the boolean and move the logic to a different function. Let me know what you think.

Reply:
I think it's a reasonable trade-off. Nice :)